ReadMe.md

```diff
@@ -162,6 +162,18 @@ console.log(colors.error("this is an error"));
 
 // outputs yellow text
 console.log(colors.warn("this is a warning"));
+
+```
+
+You can also combine them:
+
+```javascript
+colors.setTheme({
+  link: ['underline', 'blue']
+});
+
+// outputs underlined blue text
+console.log(colors.info('Listening on ') + colors.link('http://0.0.0.0:' + port)  );
```

Comment:
This did work originally for me. I'm not sure what happened, but I think we have it resolved with PR #129